### DISCUSSION TITLE
Can you help me make a FreshRSS website GDPR-compliant?

### DISCUSSION THREAD

**Discussion by mariobreskic** [FreshRSS/FreshRSS#5101] [Q&A] [answered]
Hello everyone involved in this project,
hello community of coders and users.

I plan on running a FreshRSS on a dedicated tld endling in .de, but for that, I need to be able to do the following things:

1.  I need to be able to add a link to an Impressum, that is, a page which displays, among other things, name of the website operator, an address, and contact information, and
2. if possible, use FreshRSS as a sort-of Content Management System for that Impressum page.

In short: I need to add a custom link to a custom page, ideally on all web pages of FreshRSS so the custom page can always be reached, in compliance with GDPR.

**Comment by Alkarex** [comment, marked as answer]
Hello,
This sounds like a job for a FreshRSS extension. We have various extension hooks to add custom features, and we can add more hooks if any are missing.

**Reply by mariobreskic** [reply]
Hi,
I don't see any FreshRSS extension which could help me with this. Am I missing something?

**Reply by Alkarex** [reply]
The extension would have to be written 😉. My point was that if any extensibility feature is missing, we can consider adding them.

**Reply by mariobreskic** [reply]
I see. I will look into a workaround I can work with, in some capacity. In general, is it safe to edit a docker image's content? Because I think that that is what I will end up having to do if a javascript workaround does not work?

**Reply by Alkarex** [reply]
Your edits to the Docker's image content will be lost at every updates, except if you mount your changes as Docker volumes

**Reply by mariobreskic** [reply]
My workaround using the CustomJS and Custom CSS extensions worked. I ended up adding a link to an imprint page, and I wrote the imprint page by simply saving the about page, editing its content and uploading this file to the server. Anyone interested can see the archived outcome here [archive.is](https://archive.is/skw9o) and here [archive.org](https://web.archive.org/web/20230302104921/https://grafikdesignfeed.de/i/), for posterity, and live here [https://grafikdesignfeed.de/i/](https://grafikdesignfeed.de/i/)
